bot.py

```
@@ -1,36 +1,75 @@
-# Run the bot
-if __name__ == "__main__":
-    token = os.getenv('DISCORD_TOKEN')
-    if not token:
-        raise ValueError("No token found. Make sure DISCORD_TOKEN is set in your environment variables.")
-    bot.run(token)
+import websocket
+import json
+import threading
+import time
+
+BOT_TOKEN = "REMPLACE_PAR_TON_TOKEN"
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "BOT_TOKEN"

(S105)